build/cqs.tsv

```diff
@@ -8,10 +8,10 @@ com.clearspring.analytics:stream	2.9.2	compile
 com.conversantmedia:disruptor	1.2.13	compile
 com.esotericsoftware:kryo-shaded	3.0.3	compile
 com.esotericsoftware:minlog	1.3.0	compile
-com.fasterxml.jackson.core:jackson-annotations	2.9.10	compile
-com.fasterxml.jackson.core:jackson-core	2.9.10	compile
-com.fasterxml.jackson.core:jackson-databind	2.9.10.4	compile
-com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.9.10	compile
+com.fasterxml.jackson.core:jackson-annotations	2.12.1	compile
+com.fasterxml.jackson.core:jackson-core	2.12.1	compile
+com.fasterxml.jackson.core:jackson-databind	2.12.1	compile
+com.fasterxml.jackson.dataformat:jackson-dataformat-cbor	2.12.1	compile
```

Comment:
Have you had a chance to check these at ClearlyDefined.io yet?

Reply:
(That'll let us know if actually need any CQs...)

Reply:
no, wanted to double check with you about the version changes first.

Reply:
i don't see these versions in clearly defined